examples/segmentation/example_supervoxels.cpp

```diff
@@ -231,7 +231,7 @@ main (int argc, char ** argv)
         }
         
         uint32_t rgb = static_cast<uint32_t>(color_pixel[0]) << 16 |  static_cast<uint32_t>(color_pixel[1]) << 8 |  static_cast<uint32_t>(color_pixel[2]);
-        new_point.rgb = *reinterpret_cast<float*> (&rgb);
```

Comment:
I think this piece of code has a problem. The `rgb` variable has its upper bits set to zeros, so `new_point.rgb` upper bits are zero as well. `new_point` is actually a `PointXYZRGBA` point, so the upper bits **will** be considered e.g. during visualization. And, surprise-surprise, the point will not show up, because zero Alpha means fully transparent.

I think we should rather assign color components of `new_point` individually.